node/pkg/driver/node_utils.go

```diff
@@ -320,6 +328,74 @@ func (n NodeUtils) MakeFile(filePath string) error {
 	return nil
 }
 
+func (n NodeUtils) ExpandFilesystem(devicePath string, fsType string) error {
+	var cmd string
+	var args []string
+	if fsType == "ext4" {
+		cmd = "resize2fs"
+		args = []string{devicePath}
+	} else if fsType == "xfs" {
+		cmd = "xfs_growfs"
+		args = []string{"-d", devicePath}
+	} else {
+		logger.Warningf("Skipping resize of unsupported fsType: %v", fsType)
+		return nil
+	}
+
+	logger.Debugf("Resizing the device with fs_type = {%v}", fsType)
```

Comment:
I would advice to add the devicePath to the log

Reply:
it is already logged in the next line (inside `ExecuteWithTimeout`), but I added it anyway